### DISCUSSION TITLE
Flow does not see my code, instrumentation seems to do nothing

### DISCUSSION THREAD

**Discussion by holyjak** [jpmonettas/flow-storm-debugger#24] [Q&A] [answered]
Hello! I am just trying Flow 2.2. with my lein-based project and having some issues with it. 

I have added a `#trace` in front of a function and evaluated it and then `#rtrace` in front of a expression that I then evaluated. Then:

1. A `flow-0` opens with the expression but nothing else and next to the navigation buttons there is `0/-`. 
2. When I go to Browser I see _Instrumentations_ is empty. If I find the target fn manually and click the [Instrument] button on it, in the REPL I get "Couldn't find source for <my ns>/<my fn name>". Re-running the r-traced expression opens the non-flow tob tab with a `thread-77` sub-tab with the code of the target function but agai njust `0/-` and not more data anywhere.

Any idea what could I be doing wrong? Thank you!

**Comment by jpmonettas** [comment, marked as answer]
Hi @holyjak ! 

Do you have a link to the code you are trying to instrument, or maybe you can paste here some forms?
Does the basic example on the Readme works for you?

Seeing `0/-` is expected behavior if you haven't made the first step, when you advance the debugger one step it should show the amount of traces you got for that thread.

**Reply by holyjak** [reply]
Sorry, I was wrong! I did not understood that I have to press the ▶️ button to start seeing stuf. I expected to see `0/<a positive int>` or perhaps even the first (0th?) trace. Thank you!

**Reply by jpmonettas** [reply]
Great! no problem. 
Yeah I agree that is confusing. I left the total traces like that for now just for performance reasons. 
Just modifying that javafx Label on every new trace was making things slow when I was getting 50 million traces at 200k traces per second. But will revisit that soon.
If you have any other issues I just created the #flow-storm slack channel on clojurians.
